docs/UserGuide.md

```diff
@@ -444,7 +448,11 @@ Examples:<br>
 #### Match specified property to buyers: `matchprop`
 
 Intelligently matches a property in the database to all buyers who might be interested in that property.
-These buyers must be able to accept the property's price and desire at least one of the property's characteristics.
+The property's price must be within the buyer's price range and the buyer should desire at least one of the property's characteristics.
+
+<div markdown="span" class="alert alert-primary">:exclamation: **Note:**
+Any buyers with price range more than the property's price will be ignored since the match command looks for the most suitable buyers for the property, instead of all buyers that can afford the property.
```

Comment:
```suggestion
Any buyers with price range higher than the property's price will be ignored since the match command looks for the most suitable buyers for the property, instead of all buyers that can afford the property.
```